File: MachineProblems/Code/MP6.java
import java.util.Scanner;

public class MP6 {

  public static void main(String[] args) {

    int highest = 0;
    int avg;
    int lowest = 101;
    int passed = 0;
    int total = 0;

  	int[] grades = new int[10];

    Scanner input = new Scanner(System.in);

    for (int i = 0; i < grades.length; i++) {

        grades[i] = input.nextInt();

        if (grades[i] > highest) {
            highest = grades[i];
        }

        if (grades[i] < lowest) {
            lowest = grades[i];
        }

        if (grades[i] >= 75){
            passed++;
        }
  
    }

    for (int i = 0; i < grades.length; i++) {
        total += grades[i];
    }

    avg = total / grades.length;

    System.out.println("Average Grade: " + avg);
    System.out.println("Highest Grade: " + highest);
    System.out.println("Lowest Grade: " + lowest);
    System.out.println("Number of Students Passed: " + passed);

  }
  
}

File: MachineProblems/Code/MP6.py

grades = []
highestGrade = 0
lowestGrade = 0
numOfStudentsPassed = 0

for i in range(10):

    grade = int(input())
    grades.append(grade)

    if highestGrade < grade:
        highestGrade = grade
    
    if lowestGrade >= grade or lowestGrade == 0: 
        lowestGrade = grade
    
    if grade >= 75:
        numOfStudentsPassed+=1

averageGrade = sum(grades) / len(grades)

print(f"Average Grade: {int(averageGrade)}")
print(f"Highest Grade: {highestGrade}")
print(f"Lowest Grade: {lowestGrade}")
print(f"Number of Students Passed: {numOfStudentsPassed}")
